src/server/components/dashboard/__tests__/controllers.test.ts

```diff
@@ -371,364 +341,7 @@ describe("Dashboard Controllers", () => {
     });
   });
 
-  describe.skip("listsEditController", () => {
```

Comment:
I assume it wasn't skipped by accident. It had been like that for some time. If it hasn't been touched, just delete it. It only creates more churn for you in future because you don't know wtf is going on, and it's diverged significantly from the src code. It will just be easier to rewrite them at this stage.